sort/RUNSORT.BAT

```diff
@@ -1,42 +1,79 @@
+CALL bat\ASMLG %z_TraceMode% sort\TESTSRT1 %1 %2 %3 %4 %5
+IF ERRORLEVEL 0 (echo **** No errors in TESTSRT1 21 RECORD GEN
+         ) ELSE (echo **** ++ Error! in TESTSRT1 21 RECORD GEN
+                 set  z_ReturnCode=8)
+rem
 SET SORTIN=sort\TESTSRT1.IN[RECFM=F,LRECL=20]
 SET SORTOUT=sort\TESTSRT1.OUT
 SET SYSIN=sort\TESTSRT1.INI
-erase sort\TESTSRT1.OUT
-CALL bat\EXEC sort\SORT STATS %1 %2 %3 %4 %5
-IF ERRORLEVEL 1 PAUSE ERROR IN SORT2 FOR TESTSRT1.OUT 
-copy sort\SORT.STA sort\SORT2.STA
+if exist sort\TESTSRT1.OUT erase sort\TESTSRT1.OUT
+CALL bat\EXEC %z_TraceMode% linklib\SORT STATS %1 %2 %3 %4 %5
+IF ERRORLEVEL 0 (echo **** No errors in SORT2 for TESTSRT1.OUT
+         ) ELSE (echo **** ++ Error! in SORT2 for TESTSRT1.OUT
+                 set  z_ReturnCode=8)
+move linklib\SORT.STA sort\SORT2.STA >NUL
+rem
 SET SYSUT1=sort\TESTSRT1.OUT
-CALL bat\ASMLG sort\TESTSRT2 %1 %2 %3 %4 %5
-IF ERRORLEVEL 1 PAUSE ERROR IN TESTSRT2 VERIFY TESTSRT1.OUT
-REM TESTSRT3/4 TEST SORTING 100000 USING 1000 REC TABLE
+CALL bat\ASMLG %z_TraceMode% sort\TESTSRT2 %1 %2 %3 %4 %5
+IF ERRORLEVEL 0 (echo **** No errors in TESTSRT2 verify TESTSRT1.OUT
+         ) ELSE (echo **** ++ Error! in TESTSRT2 verify TESTSRT1.OUT
+                 set  z_ReturnCode=8)
+rem
+rem TESTSRT3/4 TEST SORTING 100000 USING 1000 REC TABLE
 SET SYSUT2=sort\TESTSRT3.IN
-CALL bat\ASMLG sort\TESTSRT3 PARM(100000) %1 %2 %3 %4 %5
-IF ERRORLEVEL 1 PAUSE ERROR IN TESTSRT3 100000 RECORD GEN
+CALL bat\ASMLG %z_TraceMode% sort\TESTSRT3 PARM(100000) %1 %2 %3 %4 %5
+IF ERRORLEVEL 0 (echo **** No errors in TESTSRT3 100000 record gen
+         ) ELSE (echo **** ++ Error! in TESTSRT3 100000 record gen
+                 set  z_ReturnCode=8)
+rem
 SET SORTIN=sort\TESTSRT3.IN[RECFM=F,LRECL=4]
 SET SORTOUT=sort\TESTSRT3.OUT
 SET SYSIN=sort\TESTSRT3.INI
-erase sort\TESTSRT3.OUT
-CALL bat\EXEC sort\SORT STATS PARM(1000) %1 %2 %3 %4 %5
-IF ERRORLEVEL 1 PAUSE ERROR IN SORT FOR TESTSRT3.OUT
-copy sort\SORT.STA sort\SORT3.STA
+if exist sort\TESTSRT3.OUT erase sort\TESTSRT3.OUT
+CALL bat\EXEC %z_TraceMode% linklib\SORT STATS PARM(1000) %1 %2 %3 %4 %5
+IF ERRORLEVEL 0 (echo **** No errors in SORT for TESTSRT3.OUT
+         ) ELSE (echo **** ++ Error! in SORT for TESTSRT3.OUT
+                 set  z_ReturnCode=8)
+move linklib\SORT.STA sort\SORT3.STA >NUL
+rem
 SET SYSUT1=sort\TESTSRT3.OUT
-CALL bat\ASMLG sort\TESTSRT4 PARM(100000) %1 %2 %3 %4 %5
-IF ERRORLEVEL 1 PAUSE ERROR IN TESTSRT4 VERIFY TESTSRT3.OUT
-PAUSE VERIFY ALL SORT TESTS OK
-
+CALL bat\ASMLG %z_TraceMode% sort\TESTSRT4 PARM(100000) %1 %2 %3 %4 %5
+IF ERRORLEVEL 0 (echo **** No errors in TESTSRT4 verify TESTSRT3.OUT
+         ) ELSE (echo **** ++ Error! in TESTSRT4 verify TESTSRT3.OUT
+                 set  z_ReturnCode=8)
+rem
+if exist SORTWK*.TMP erase SORTWK*.TMP
+if %z_ReturnCode% == 0 echo **** All SORT tests completed okay
 
+:return
+rem ----- Lvl(%z_NestLevel%)  End %0 %1 %2 %3 %4 %5 %6 %7 %8 %9
+exit /b %z_ReturnCode%
```

Comment:
sort/RUNSORT.BAT is obsolete. Replaced by rt/bat/RUNSORT.BAT and rt/bash/runsort. Make changes there and delete sort/RUNSORT.BAT.

Reply:
Okay, I'll look into it. Thanks for notifying me - I had not noticed.

Reply:
It appears that you've decided to keep the obsolete sort/RUNSORT.BAT. Why?